src/cheri/riscv-unpriv-integration.adoc

```diff
@@ -900,3 +898,15 @@ NOTE: Future behavior for these reserved branch encodings may include branching
 If the target of a taken branch lies outside the bounds of <<pcc>>, the next instruction fetch will raise an exception.
 
 NOTE: A future extension may raise an exception on the branch instruction itself if fetching a minimum sized instruction at the target <<pcc>> will raise a _{cheri_excep_name_pc}_.
+
+
+== "{rvy_sentry_ext_name}" Extension for Ambient Creation of Sentry Capabilities
+
+In the {cheri_base_ext_name} base ISA the only way to create a <<sentry_cap>> is through the <<JALR_CHERI>> instruction as that writes back a sentry capability to the link register.
+The {rvy_sentry_ext_name} extension adds the <<SENTRY>> instruction to allow sealing arbitrary capabilities as a <<sentry_cap>> (<<sec_cap_type>>=1).
+
+NOTE: Future extensions to {cheri_base_ext_name} (such as CHERIoT) support additional <<sec_cap_type>> values as alternatives to the ambient <<sentry_cap>> (<<sec_cap_type>>=1), so this instruction is part of an optional extension instead of the {cheri_base_ext_name} base ISA.
```

Comment:
maybe an xref to uycfg which is the only other reference to cheriot in the spec right now?

Reply:
done.